src/api/index.js

```diff
@@ -49,6 +49,30 @@ const updatePromiseUpdate = data => updateSomething(`${PROMISE_UPDATES_PATH}${da
 
 const getGeneralStats = () => getSomething('/stats/general_stats')
 
+const mockLists = [
+  {
+    id: 'm54ertdfgcv',
+    title: 'Student Aids',
+    promise_ids: [
+      'CeX1MLvSq7IVx3nI1CgI',
+      'U5zwNgRbdu0NZRjFLoBF',
+      'o3lyMjqZrCALP6HSUZFS'
+    ]
+  },
+  {
+    id: 'ytughjbsdf8t7yiu',
+    title: 'Broadband service in Malaysia',
+    promise_ids: [
+      'XjGLsRtJ9spk5VeBidDl',
+      'bGvbmCPqIcpWQIZAp92C'
+    ]
+  }
+]
+
+const listLists = () => mockLists
```

Comment:
i just thought about a better naming convention, e.g. `getMockList()` WDYT?

Reply:
the idea is, once the API endpoint is ready, is to then replace `mockLists` with `getSomething()` call. check the other examples.
the interface, i.e. api.listLists should not be changed. the component that calls it (Lists) should not need to change the method it is calling

Reply:
oh, yeah, i just saw the other fully implementation:
`const listContributors = () => getSomething('/contributors/')`

sorry @nazreen 🙏